### PR TITLE
jupyter: add page

### PR DESCRIPTION
**Pull request by ericchang00**
----

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines

### CODE REVIEW

File: pages/common/jupyter.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# jupyter
+
+> Web application for scientific computing and reproducible research.
```
Comment (agnivade) [inline pages/common/jupyter.md]: This does not mention the primary purpose of jupyter, which is to share documents containing code, equations etc. How about 

```
> Web application to create and share documents that contain code, visualizations and notes.
> Primarily used for scientific computing and machine learning.
``` 
?

**Comment by jsonbruce** [comment]
We can open a particular notebook from command line via
`jupyter notebook example.ipynb`

@agnivade @ericchang00

**Comment by agnivade** [comment]
Great. Let's add it.

**Comment by ericchang00** [comment]
Thanks for the input and the heads up! My apologies for letting this fall off my radar.

**Comment by agnivade** [comment]
Travis has stopped all open source builds. https://www.traviscistatus.com/incidents/c1s4dlyxd4lf. Merging this forcefully.

**Pull request merged**